sdk-dotnet/LittleHorse.Sdk/Internal/LHOptionsBinder.cs

```diff
@@ -52,5 +51,60 @@ internal static LHOptions GetOptionsFromEnvironmentVariables()
 
             return options;
         }
+
+        internal static LHOptions GetOptionsFromFile(string filePath)
+        {
+            var options = new LHOptions();
+
+            Dictionary<string, string> lHOptionsInFile = FileManager.GetDictFromPath(filePath);
```

Comment:
I see that `FileManager` class is included into a `utils` package, but I don't think this is a utility class. I think `GetDictFromPath` should be a private method inside LHOptions

Reply:
In this case is a general implementation that can be reused for any case where it is neede to get a dict from a file path that is the reason of beign a utility, it does not have LHOptions logic

Reply:
LHOptions is the only class that can read a property file. Where do we want to reuse this logic?

Reply:
Is there any reason because we have LHOptions?

Reply:
it is a class where all the options for the LH configuration are placed, I think if we join options in LHConfig the file will be big